universe/syncer.go

```diff
@@ -74,14 +74,39 @@ func (s *SimpleSyncer) executeSync(ctx context.Context, diffEngine DiffEngine,
 		s.isSyncing.Store(false)
 	}()
 
+	// If we're not syncing a specific set of Universes, then we'll
+	// sync all Universes which have an associated specific sync config.
+	// These universes are filtered based on the actual config content
+	// later.
+	if len(idsToSync) == 0 {
+		for _, uniSyncCfg := range syncConfigs.UniSyncConfigs {
+			idsToSync = append(idsToSync, uniSyncCfg.UniverseID)
+		}
+	}
+
+	// Filter out any universe IDs that we don't want to sync based on the
+	// content of the federation sync config.
+	idsToSync = fn.Filter(
+		idsToSync, func(id Identifier) bool {
+			// If we're syncing issuance proofs, then we'll only
+			// sync issuance roots.
+			if syncType == SyncIssuance &&
+				id.ProofType != ProofTypeIssuance {
+
+				return false
+			}
+
+			return syncConfigs.IsSyncInsertEnabled(id)
```

Comment:
nit (for another PR): `IsSyncInsertEnabled()` loops over all configs we have; maybe those configs should be a `Set` instead of a list if we need to check membership often.